packages/core-utils/src/__tests__/route.js

```diff
@@ -135,4 +136,60 @@ describe("util > route", () => {
       expect(sortRoutes(route12, route13)).toMatchSnapshot();
     });
   });
+
+  describe("route text color generation", () => {
```

Comment:
Maybe add a comment about where these color combinations and expected values come from.

Reply:
Better idea, could you replace this test suite into a visual in storybook? (Do a "before" and "after" comparison).

Reply:
I've added an interactive way to test the method.